go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -586,12 +585,14 @@ type VRSettings struct {
 	MaxTPS            int64
 	MaxReplicationLag int64
 	State             string
+	WorkflowType      int64
```

Comment:
Nice! I was going to need to do something similar if we ever decide to move forward with https://github.com/vitessio/vitess/pull/9845 as we only want to force the PK for MoveTables.

Reply:
This was stolen from #9742 authored by @rohit-nayak-ps